### PR TITLE
added myself to the student's section

### PR DESCRIPTION
**Pull request by abhijeetmanohar**
updated my info on website....added myself to the student's section

### CODE REVIEW

**Comment by abishekvashok** [comment]
Fix the checks

File: _data/students.yml [hunk 1]
```diff
@@ -14,6 +14,13 @@
   facebook: Rosca.Oanaa
   bio: 17, Romanian, coder, geek, science enthusiast :D
 
+- name: Abhijeet Manohar
+  github: abhijeetmanohar
+  image: Abhijeet.jpg
+  twitter: abhijeetmanohar
+  facebook: abhijeetmanohar66
+  bio: 16, Indian, code-lover, roboticgeek
```
Comment (jigyasa-grover) [inline _data/students.yml]: Trailing whitespace at `LINE 22`

Remove it to fix build failure.

**Comment by jigyasa-grover** [comment]
* Rename the commit to something meaningful, such as "Adding <-your-name-> in students/mentors section"
* Also provide the link to live `gh-pages` from your forked repository or relevant screenshot for easier review.

**Comment by rhemon** [comment]
you still gotta fix to pass the tests

**Comment by rhemon** [comment]
its failing because of white spaces

**Comment by abishekvashok** [comment]
and there are conflicts

**Comment by abhijeetmanohar** [comment]
https://abhijeetmanohar.github.io/gci16.fossasia.org/

**Comment by abishekvashok** [comment]
I closed an opened so that the tests would restart. @abhijeetmanohar

**Comment by m1guelpf** [comment]
@Abhi2424shek You can also go to Travis and hit restart build...

**Comment by abishekvashok** [comment]
Yes next time see me hitting that button @m1guelpf

**Comment by niccokunzmann** [comment]
Some of the travis build errors were already fixed.

**Comment by niccokunzmann** [comment]
Thanks for your contribution!

**Pull request merged**